src/main/java/graphql/nadel/engine/ServiceResultToResultNodes.java

```diff
@@ -106,7 +106,7 @@ private Object fetchValue(Object source, String key) {
         return map.get(key);
     }
 
-    private ExecutionResultNode analyseValue(ExecutionContext executionContext,
+    private ExecutionResultNode analyzeValue(ExecutionContext executionContext,
```

Comment:
> Analyse" is much more common in British English and "analyze" is the American and Canadian spelling of the same word. 

;)   I dont care either way

Reply:
I personally prefer the British version (it's called English and not American after all...) BUT I see everywhere else in Nadel we have gone for the American spelling.

Reply:
yes we use the US spelling